v3/mqtt.go

```diff
@@ -852,6 +917,25 @@ func (conn *mqttConn) getResponseChannel(pktType packet.Type) chan MqttResponse
 	}
 }
 
+func (conn *mqttConn) sendQueueingError(err error) {
+	if err == nil {
+		err = errors.New("Channel full. Sending on error channel")
```

Comment:
Go langers are like an early return. I think.

```suggestion
		return errors.New("Channel full. Sending on error channel")
```

Reply:
Oh, I'm not returning the error, I'm creating a new error and writing to the channel (or adding to a slice).

Reply:
Oh!!! Sorry! It is my miss-reading!